pairtools/lib/stats.py

```diff
@@ -739,6 +787,19 @@ def add_pairs_from_dataframe(self, df, unmapped_chrom="!"):
             self._stat[key]["total_single_sided_mapped"] += int(
                 total_count - (mapped_count + unmapped_count)
             )
+            if self.single_mapped_by_side:
+                left_only_mapped_count = np.logical_and(
+                    df_filtered["chrom1"] != unmapped_chrom,
```

Comment:
Hmmm, this definition of "one-sided mapped" won't work for chimeric reads originating from R1. It will count them as "two-sided" although they factually come from R1 oly. This looks like a re-implementation of single-end mapped pairs.

Reply:
Ummm is there a way to account for those properly at all at this stage? I think without some extra arguments this information is just lost...

Reply:
Yeah, you need walk_pair_type column for that
https://pairtools.readthedocs.io/en/latest/parsing.html#rescuing-complex-walks section walk_pair_types